rest/src/main/java/com/netflix/conductor/rest/rbac/RbacHttpFilter.java

```diff
@@ -73,9 +73,11 @@ public void doFilter(
 
         HttpServletRequest request = (HttpServletRequest) servletRequest;
         HttpServletResponse response = (HttpServletResponse) servletResponse;
+        List<String> noRbacValues = List.of("/health", "/v3/api-docs");
 
-        final String healthCheck = "health";
-        if (request.getRequestURI().contains(healthCheck)) {
+        boolean isRbacless =
+                noRbacValues.stream().anyMatch(value -> value.equals(request.getRequestURI()));
```

Comment:
Change to startWith() for capture sub-requests or query parameters.

Reply:
Resolved.